CMakeLists.txt

```diff
@@ -1511,59 +1511,62 @@ if(BUILD_TESTS)
       )
     endif()
 
-    add_piccolo_test(
-      NAME pi_ls
-      PYTHON_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/tests/infra/piccolo_driver.py
-      CLIENT_BIN ./submit PERF_LABEL "Logging"
-      ADDITIONAL_ARGS --package "samples/apps/logging/liblogging"
-                      --max-writes-ahead 1000 --repetitions 10000
-    )
-
-    add_piccolo_test(
-      NAME pi_basic
-      PYTHON_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/tests/infra/basicperf.py
-      CLIENT_BIN ./submit PERF_LABEL "Basic"
-      ADDITIONAL_ARGS --package "samples/apps/basic/libbasic" --client-def
-                      "1,write,100000,primary"
-    )
-
-    add_piccolo_test(
-      NAME pi_basic_js
-      PYTHON_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/tests/infra/basicperf.py
-      CLIENT_BIN ./submit PERF_LABEL "Basic JS"
-      ADDITIONAL_ARGS --js-app-bundle ${CMAKE_SOURCE_DIR}/samples/apps/basic/js
-                      --client-def "1,write,100000,primary"
-    )
+    if(CMAKE_BUILD_TYPE STREQUAL "Release")
```

Comment:
Probably not what we want (our default is not Release, it's RelWithDebInfo anyway), proposed an alternative solution under https://github.com/microsoft/CCF/issues/6421#issuecomment-2268456893.

Reply:
I don't understand the rationale. I think we should remove the -LE, just like we shouldn't have it for the partitions tests. It's weird to have code that does nothing just in case one specific type of code change happens.

Reply:
`-LE perf` means "I wanna run all but perf"
`-L ...|perf|... -C e2e_perf` means "I want to run e2e perf tests in the selected test suite.

The `e2e_perf` configuration can be treated as a subset of perf tests in this case.

Is it still weird if we read it like that?

Reply:
It's not weird but afaict e2e_perf _is_ perf?

Reply:
Convincing. Done